src/hotspot/share/gc/shared/cardTable.cpp

```diff
@@ -26,13 +26,40 @@
 #include "gc/shared/cardTable.hpp"
 #include "gc/shared/collectedHeap.hpp"
 #include "gc/shared/space.inline.hpp"
+#include "gc/shared/gcLogPrecious.hpp"
+#include "gc/parallel/objectStartArray.hpp"
 #include "logging/log.hpp"
 #include "memory/virtualspace.hpp"
 #include "runtime/java.hpp"
 #include "runtime/os.hpp"
+#include "runtime/globals_extension.hpp"
```

Comment:
I think we typically include the globals include that contains the flag - this should probably be `#include "gc/shared/gc_globals.hpp` (and sorted correctly in this list).